install.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+home=$HOME
+echo "██╗    ██╗██╗    ██╗ ██████╗████████╗██╗   ██╗██╗███╗   ███╗"
+echo "██║    ██║██║    ██║██╔════╝╚══██╔══╝██║   ██║██║████╗ ████║"
+echo "██║ █╗ ██║██║ █╗ ██║██║        ██║   ██║   ██║██║██╔████╔██║"
+echo "██║███╗██║██║███╗██║██║        ██║   ╚██╗ ██╔╝██║██║╚██╔╝██║"
+echo "╚███╔███╔╝╚███╔███╔╝╚██████╗   ██║    ╚████╔╝ ██║██║ ╚═╝ ██║"
+echo " ╚══╝╚══╝  ╚══╝╚══╝  ╚═════╝   ╚═╝     ╚═══╝  ╚═╝╚═╝     ╚═╝"
+
+exist=1
+if command -v yay &> /dev/null; then
+  yay -R --noconfirm neovim
+  yay -S --noconfirm neovim nvim-packer-git
```

Comment:
This command will try to install neovim twice. We also don't need packer anymore so please remove this line